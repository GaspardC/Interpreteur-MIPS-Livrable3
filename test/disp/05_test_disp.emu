#-------
#TEST_RETURN_CODE=FAIL           
#TEST_COMMENT="Teste fonctionnement standard"
#-------

disp reg $1 $4 $afds
