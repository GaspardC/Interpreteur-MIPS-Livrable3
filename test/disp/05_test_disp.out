at : 0 a0 : 0 probl√®me dans l'affichage du nom du registre : ffffffff 